AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -50,14 +50,15 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId flags:(MSFlags)fl
     return NO;
   }
   MSFlags persistenceFlags = flags & kMSPersistenceFlagsMask;
+  long long timestampMs = (long long)([log.timestamp timeIntervalSince1970] * 1000);
```

Comment:
We already have a similar method in MSUtility. This can be placed there too.

Reply:
@achocron tried to address the feedback but couldn't see any reason of this one, I also agreed.